bugs/sru-19.4.33-manual-4.txt

```diff
@@ -0,0 +1,306 @@
+=== Begin SRU Template ===
+[Impact]
+Images which provide multiple ssh authorized key files for users via
+setting AuthorizedKeysFiles = <file1> <file2> would result in an invalid ssh
+configuration resulting in no remote ssh connectivity due to the absence of
+any .ssh/authorized_keys file.
+
+In this scenario, old/broken cloud-init would emit
+a separate subdirectory that would represent the space-delimited string value
+as a single subdirectory representing the full string instead of consolidating
+the keys from the separate AuthorizedKeysFiles represented in the
+/etc/ssh/sshd_config AuthorizedKeysFiles value.
+
+[Test Case]
+Create an lxc instance with AuthorizedKeyFiles set to mulitple files.
+.ssh/authorized_keys will contain one pre-existing key.
+/default-keys will contain a one pre-existing default ssh pubkey.
+Launch an lxc vm providing a third ssh pubkey via ssh_import_id.
+Ensure all three keys are present in .ssh/authorized_keys.
+
+#!/bin/sh
+set -x
+
+# To be adapted to the SRU to test
+SRU_SERIES="xenial bionic eoan"
+
+cat > importme.yaml <<EOF
+## template: jinja
+#cloud-config
+ssh_import_id : [chad.smith]
+EOF
+
+cat > setup_ssh_multi_keys.sh <<EOF
+sed -i 's/#AuthorizedKeysFile.*/AuthorizedKeysFile	%h\/\.ssh\/authorized_keys \/default_keys/' /etc/ssh/sshd_config
```

Comment:
I find it much easier to read path substitutions if you use something other than `/` for the sed delimiter:

```suggestion
sed -i 's,#AuthorizedKeysFile.*,AuthorizedKeysFile	%h/.ssh/authorized_keys /default_keys,' /etc/ssh/sshd_config
```

Reply:
Nice, Thanks TIL I could replace that delimiter